src/org/ods/orchestration/util/Project.groovy

```diff
@@ -1310,6 +1314,42 @@ class Project {
         return result
     }
 
+    protected Map loadJiraDataSecurityVulnerabilities(String versionName = null) {
+        if (!this.jiraUseCase) return [:]
+        if (!this.jiraUseCase.jira) return [:]
+
+        def fields = ['assignee', 'duedate', 'issuelinks', 'status', 'summary']
+        def jql = "project = ${this.jiraProjectKey} AND issuetype = 'Security Vulnerability' AND status != Done"
+
+        if (versionName) {
+            fields << 'fixVersions'
+            jql = jql + " AND fixVersion = '${versionName}'"
+        }
+
+        def jqlQuery = [
+            fields: fields,
+            jql: jql,
+            expand: []
+        ]
+
+        def securityVulnerabilities = this.jiraUseCase.jira.getIssuesForJQLQuery(jqlQuery) ?: []
+
+        return securityVulnerabilities.collectEntries { secVul ->
+            def issue = [
+                key: secVul.key,
+                name: secVul.fields.summary,
+                assignee: secVul.fields.assignee ?
+                    [secVul.fields.assignee.displayName,
+                     secVul.fields.assignee.name,
+                     secVul.fields.assignee.emailAddress].find { it != null } : "Unassigned",
+                dueDate: '',
```

Comment:
Why are obtaining due date if is not used here.

Reply:
Removed.